internal/merkle/memory/memory_merkle_tree.go

```diff
@@ -12,10 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package merkle
-
-// This is a fairly direct port of the C++ Merkle Tree. It has the same API and should have
-// similar performance. It keeps all its data in RAM and is not part of the Trillian API.
+// Package memory provides a fairly direct port of the C++ Merkle Tree from the CT repo.
```

Comment:
TODO: Describe the type, instead of referencing to C++ repo.

Reply:
done